audio/mididrv.cpp

```diff
@@ -145,7 +145,7 @@ MusicType MidiDriver::getMusicType(MidiDriver::DeviceHandle handle) {
 	return MT_INVALID;
 }
 
-Common::String MidiDriver::getDeviceString(DeviceHandle handle, DeviceStringType type) {
+Common::U32String MidiDriver::getDeviceString(DeviceHandle handle, DeviceStringType type) {
```

Comment:
This should be kept at String, the whole file except the places which use `_()`. No need to return U32String where there is ASCII-only